sick-fits/backend/.vscode/settings.json

```diff
@@ -12,9 +12,20 @@
   "[javascriptreact]": {
     "editor.formatOnSave": false
   },
+  "[typescript]": {
+    "editor.formatOnSave": false
+  },
+  "[typescriptreact]": {
+    "editor.formatOnSave": false
+  },
```

Comment:
these settings were not necessary for me to get prettier and eslint working. I believe 👇 does what we need, but maybe it's safer to disable the format on save 🤷 
```
"editor.codeActionsOnSave": {
    "source.fixAll": true
    ...
```